state-chain/pallets/cf-asset-balances/src/lib.rs

```diff
@@ -128,6 +146,25 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type WithheldAssets<T: Config> =
 		StorageMap<_, Twox64Concat, Asset, AssetAmount, ValueQuery>;
+
+	#[pallet::storage]
+	/// Storage for user's free balances/ DoubleMap: (AccountId, Asset) => Balance
+	pub type FreeBalances<T: Config> =
+		StorageDoubleMap<_, Twox64Concat, T::AccountId, Identity, Asset, AssetAmount>;
+
+	#[pallet::storage]
+	/// Historical earned fees for an account.
+	pub type HistoricalEarnedFees<T: Config> =
+		StorageDoubleMap<_, Identity, T::AccountId, Twox64Concat, Asset, AssetAmount, ValueQuery>;
+
+	/// Network fees, in USDC terms, that have been collected and are ready to be converted to FLIP.
+	#[pallet::storage]
+	pub type CollectedNetworkFee<T: Config> = StorageValue<_, AssetAmount, ValueQuery>;
```

Comment:
This was recently (in https://github.com/chainflip-io/chainflip-backend/pull/5014) moved from LP to the Swapping pallet (as per [PRO-1404](https://linear.app/chainflip/issue/PRO-1404)). The motivation was to simplify code by avoiding indirection via traits. 

I'm surprised we want to move this again. Looking at https://linear.app/chainflip/issue/PRO-1422/move-asset-balances-into-dedicated-pallet, it seems like the plan was to move it to the swapping pallet rather than a new pallet (which has already been done)?

Also, doesn't look like this PR does not remove `CollectedNetworkFee` from its previous location. I'm guessing it is a merge error?

Reply:
I guess we don't then. I was not aware that this was the plan, but like it seems we gonna keep it in the swapping pallet in this case.

Yeah, that was due to merge error.